src/components/show-edit-pod/show-edit-pod.component.js

```diff
@@ -10,11 +10,14 @@ import Events from "../../__internal__/utils/helpers/events";
 import tagComponent from "../../__internal__/utils/helpers/tags/tags";
 import LocaleContext from "../../__internal__/i18n-context";
 import StyledPod from "./show-edit-pod.style";
+import Logger from "../../__internal__/utils/logger";
 
 const marginPropTypes = filterStyledSystemMarginProps(
   styledSystemPropTypes.space
 );
 
+let deprecateInputRefWarnTriggered = false;
```

Comment:
There is no input ref

```suggestion
let deprecateWarnTriggered = false;
```